python/upload_templates.py

```diff
@@ -0,0 +1,207 @@
+import numpy as np
+import s3fs
+import zarr
+import numcodecs
+import os
+
+
+from dandi.dandiapi import DandiAPIClient
+from spikeinterface.extractors import NwbRecordingExtractor, IblSortingExtractor
+from spikeinterface.extractors import IblRecordingExtractor
+
+from one.api import ONE
+from spikeinterface.preprocessing import (
+    astype,
+    phase_shift,
+    common_reference,
+    highpass_filter,
+)
+from spikeinterface.core import create_sorting_analyzer
+
+client = DandiAPIClient.for_dandi_instance("dandi")
+
+dandiset_id = "000409"
+dandiset = client.get_dandiset(dandiset_id)
+
+
+valid_dandiset_path = lambda path: path.endswith(".nwb") and "ecephys" in path
+dandiset_paths_with_ecephys = [
+    asset.path for asset in dandiset.get_assets() if valid_dandiset_path(asset.path)
+]
+dandiset_paths_with_ecephys.sort()
+dandiset_paths_with_ecephys = [
+    path for path in dandiset_paths_with_ecephys if "KS" in path
+]
+
+
+for asset_path in dandiset_paths_with_ecephys:
+
+    asset_path = dandiset_paths_with_ecephys[3]
+    recording_asset = dandiset.get_asset_by_path(path=asset_path)
+    url = recording_asset.get_content_url(follow_redirects=True, strip_query=True)
+    file_path = url
+
+    electrical_series_path = "acquisition/ElectricalSeriesAp00"
```

Comment:
@alejoe91 I think this the 00 at the end of the path is restricting us to only one the two mounted probes? What do you think?

Reply:
Yes, maybe we can use the `find_electrical_series` and add different AP streams as different datasets

Reply:
Let me test this.